repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/ipuser/IpUserTest.groovy

```diff
@@ -89,4 +93,23 @@ class IpUserTest extends ReposeValveTest {
         !group.contains("local-lan-ip;q=0.6")
     }
 
+    def "should fail to start with bad config"() {
```

Comment:
Yeah, that's kind of where I'm at. My only concern is that, at some point in the future, we will check which filters do and do not have this bad config test, and the way we check may be to look for a certain test file. Or, more generally, forming more patterns makes searching much easier. So I vote for breaking it out, even though you're right.